package.json

```diff
@@ -2,11 +2,16 @@
   "name": "@buttercup/iconographer",
   "version": "0.1.0",
   "description": "Icon queuing, fetching and caching library",
-  "main": "source/index.js",
+  "main": "dist/index.js",
+  "files": [
+    "dist"
+  ],
   "scripts": {
+    "build": "babel -d dist source",
     "docs": "jsdoc2md source/**/*.js > API.md",
     "format": "prettier --write '{source,test}/**/*.js'",
     "precommit": "lint-staged",
+    "prepublishOnly": "npm run test && npm run format && npm run build",
     "test": "run-s test:unit test:format",
```

Comment:
Seeing as this is the target on our CI server, maybe it'd be best to include "build" in the test command so we can confirm that the build process functions correctly. In another PR, we could point the tests to the babel-built source rather than regular source.